db-client/src/index.ts

```diff
@@ -9,14 +9,23 @@ import { DB, Versioned, Q } from './db';
 
 export { Q };
 
-// TODO(bergundy): Verify environment variables
+function assertEnv(name: string): string {
```

Comment:
Better than undefineds, although still missing a test of shift-db in local-proxy (not necessarily for this PR).

Reply:
#172